renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -435,12 +435,7 @@ __attribute__((visibility("default"))) EGLContext eglCreateContext(EGLDisplay di
   init.depthBits = value;
   eglhooks.eglGetConfigAttrib_real(display, config, EGL_STENCIL_SIZE, &value);
   init.stencilBits = value;
-// TODO: how to detect this?
-#if ENABLED(RDOC_ANDROID)
   init.isSRGB = 0;
```

Comment:
Interesting, that makes more sense then. I'm not surprised a 24-bit texture isn't color-renderable, since I guess hardware will mostly want 32-bit pixel strides. Not important to fix right now but it's good to know for the future that this is soluble.